### PR TITLE
Fixes #13520 : Many UI automation tests are failing in interactive_menu/basic_options_spec.js

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
A lots of tests were failing due to unable to find the input of select of webhook messages, this pr fixes those in addition to closing all the rhs tabs

#### Ticket Link
Fixes : https://github.com/mattermost/mattermost-server/issues/13520
Jira : None

### CODE REVIEW

File: e2e/cypress/integration/interactive_menu/basic_options_spec.js [hunk 1]
```diff
@@ -191,20 +195,23 @@ describe('Interactive Menu', () => {
     });
 
     it('IM21042 - "No items match" feedback', () => {
-        const missingUser = Date.now();
+        const missingUser = '1577793817254';
```
Comment (saturninoabril) [inline e2e/cypress/integration/interactive_menu/basic_options_spec.js]: I'm fine with this but the old implementation seems to be not broken.

File: e2e/cypress/integration/interactive_menu/basic_options_spec.js [hunk 2]
```diff
@@ -191,20 +195,23 @@ describe('Interactive Menu', () => {
...
+            cy.get(`#messageAttachmentList_${postId}`).within(() => {
+                // # Type the missing user in the select input
+                cy.findByPlaceholderText('Select an option...').click().clear().type(`${missingUser}`);
 
-        cy.get('@messageAttachmentList').within(() => {
-            cy.get('.select-suggestion-container > input').click().clear().type(`${missingUser}`);
-            cy.get('.suggestion-list__no-results').should('be.visible').should('have.text', `No items match ${missingUser}`);
+                cy.get('#suggestionList').within(() => {
+                    // * Check if we get appropriate message when no options matches entered text
+                    cy.findByText(/No items match/gmui).should('exist');
```
Comment (saturninoabril) [inline e2e/cypress/integration/interactive_menu/basic_options_spec.js]: Could you please update to meet the test expectation of "No items match [what you have typed]" is displayed above the input/selection box"?

Comment (M-ZubairAhmed) [reply e2e/cypress/integration/interactive_menu/basic_options_spec.js]: Yes so if i write in template literals ``No item match ${selectes}``, tests are unable to find it. So i had to remove it. How do i write the variable in regex ?

Comment (M-ZubairAhmed) [reply e2e/cypress/integration/interactive_menu/basic_options_spec.js]: Anyways i reverted the change

**Comment by saturninoabril** [comment]
Merging this now, being a quick fix on existing test.

**Pull request merged**